tests/Fixtures/expected/references/Serializer/DateTimeDenormalizer.md

```diff
@@ -0,0 +1,67 @@
+# [PhpDocumentGenerator\Tests\Fixtures\Serializer\DateTimeDenormalizer](docs/references/Serializer/DateTimeDenormalizer) implements Symfony\Component\Serializer\Normalizer\DenormalizerInterface, Symfony\Component\Serializer\Normalizer\DenormalizerAwareInterface, [PhpDocumentGenerator\Tests\Fixtures\Services\IgnoredInterface](docs/references/Services/IgnoredInterface)
+
+## Properties:
+
+### <a href="#property-denormalizer" id="property-denormalizer">§</a> protected Symfony\Component\Serializer\Normalizer\DenormalizerInterface $denormalizer
+
+## Methods:
+
+### <a href="#method-denormalize" id="method-denormalize">§</a> public function denormalize([mixed](https://php.net/mixed) $data, [string](https://php.net/string) $type, [string](https://php.net/string) $format, [array](https://php.net/array) $context): [mixed](https://php.net/mixed)|[string](https://php.net/string)|[stdClass](https://php.net/class.stdclass)|[DateTimeInterface](https://php.net/class.datetimeinterface)[array&lt;int, DateTimeInterface&gt;](https://php.net/array&lt;int, DateTimeInterface&gt;)
+
+This method throws multiple exception and returns multiple types to ensure the types are correctly imported.
+
+Denormalizes data back into an object of the given class.
+
+Also, the parent documentation should be placed in the middle of this method documentation.
+
+#### Parameters
+
+  - [mixed](https://php.net/mixed) $data Data to restore
+  - [string](https://php.net/string) $type The expected class to instantiate
+  - [string](https://php.net/string) $format Format the given data was extracted from
+  - [array](https://php.net/array) $context Options available to the denormalizer
+
+#### Returns
+
+  - [mixed](https://php.net/mixed) the data
+  - [string](https://php.net/string)
+  - [stdClass](https://php.net/class.stdclass)
+  - [DateTimeInterface](https://php.net/class.datetimeinterface)
+  - [array&lt;int, DateTimeInterface&gt;](https://php.net/array&lt;int, DateTimeInterface&gt;)
```

Comment:
`https://php.net/array&lt;int, DateTimeInterface&gt;` has wrong encoding.

Also, the link `https://php.net/array<int, DateTimeInterface>` is invalid. Should we handle it this way instead? `[array](https://php.net/array)<[int](https://php.net/int), [DateTimeInterface](https://php.net/datetimeinterface)`